content/en/docs/tasks/tools/install-kubectl-windows.md

```diff
@@ -79,14 +79,19 @@ If you have installed Docker Desktop before, you may need to place your `PATH` e
 
 ### Install on Windows using Chocolatey or Scoop
 
-1. To install kubectl on Windows you can use either [Chocolatey](https://chocolatey.org) package manager or [Scoop](https://scoop.sh) command-line installer.
+1. To install kubectl on Windows you can use either [Winget](https://docs.microsoft.com/en-us/windows/package-manager/winget/) windows package manager or [Chocolatey](https://chocolatey.org) package manager or [Scoop](https://scoop.sh) command-line installer.
 
    {{< tabs name="kubectl_win_install" >}}
    {{% tab name="choco" %}}
    ```powershell
    choco install kubernetes-cli
    ```
    {{% /tab %}}
+   {{% tab name="winget" %}}
+   ```powershell
+   winget install -e --id Kubernetes.minikube
+   ```
+   {{% /tab %}}
```

Comment:
We should either list Winget first (it seems to be the solution that Microsoft recommends), or list Chocolatey, Scoop and then Winget (alphabetical order).

Reply:
Ok